graphs/pyproject.toml

```diff
@@ -73,6 +73,12 @@ optional-dependencies.hex = [ "h3>=3.7.7,<4" ]
 optional-dependencies.tests = [ "hypothesis>=6.11", "pytest>=8", "pytest-mock>=3" ]
 
 optional-dependencies.tri = [ "trimesh>=4.1" ]
+
+optional-dependencies.xarray = [
+  "xarray<=2025.4",
```

Comment:
is `<=2025.4` needed?

Does the code break when using xarray>2025.4 ? If yes, could you add a comment here, please?
This will save us time when we will have another package requiring `xarray>2025.4` and we have to update the code.